lib/web_ui/lib/src/engine/pointer_binding/event_position_helper.dart

```diff
@@ -23,15 +23,20 @@ import '../window.dart';
 ///
 /// It also takes into account semantics being enabled to fix the case where
 /// offsetX, offsetY == 0 (TalkBack events).
-ui.Offset computeEventOffsetToTarget(DomMouseEvent event, EngineFlutterView view) {
+ui.Offset computeEventOffsetToTarget(
+  DomMouseEvent event,
+  EngineFlutterView view, {
+  DomEventTarget? eventTarget,
```

Comment:
By the time we get to this method, it's a little bit weird that we're passing both `event` and `eventTarget`.

Can you make a note in the docs of this method that "because of how Safari coalesces events", we need to pass both the `event` (part of the batch) and the `eventTarget` (coming from the global event?)

Reply:
Yep, I'll add a note. It's not only Safari. Chrome also had this issue in v129 (see https://github.com/flutter/flutter/issues/155987).

Reply:
I'm not a W3C spec expert, but if I'm reading this right:

* https://w3c.github.io/pointerevents/#populating-and-maintaining-the-coalesced-and-predicted-events-lists

> When a trusted `PointerEvent`'s `target` is changed, user agents SHOULD, for each event in the `coalesced events list` and `predicted events list`:
>
> 1. Set the event's `target` to match the `target` of the "parent" pointer event.

it seems that Safari is **not** doing the Right Thing here because IMO it seems that the coalesced events *should* have a `target` that matches that of the parent event 😕